.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "shared-modules"]
 	path = shared-modules
 	url = https://github.com/flathub/shared-modules.git
+
```

Comment:
Since you're not including the submodule anymore, you can revert this particular change to this file.